sdcm/results_longevity.html

```diff
@@ -2,7 +2,6 @@
 {% block body %}
     <h3>Test: {{ test_name }}</h3>
     <h3>Test-id: {{ test_id }}</h3>
-    <h3>Document-id {{ document_id }}</h3>
```

Comment:
The document_id is actually the test_id, we also repeatedly reported it nemesis stat.
Do you want to remove that also?

Reply:
```
$ git grep document_id
longevity_test.py:            'document_id': self.get_doc_id()
sdcm/results_longevity.html:    <h3>Document-id {{ document_id }}</h3>
sdcm/send_email.py:              'nemesis_name', 'nemesis_details', 'test_id', 'document_id']
sdcm/send_email.py:              'build_url', 'nemesis_name', 'nemesis_details', 'test_id', 'document_id']
[amos@fedora sct-master]$ git grep get_doc_id
longevity_test.py:            'document_id': self.get_doc_id()
sdcm/db_stats.py:    def get_doc_id(self):
sdcm/db_stats.py:            result = self.elasticsearch.get_doc(self._test_index, self.get_doc_id(), doc_type=self._es_doc_type)
[amos@fedora sct-master]$ git grep get_doc_data
sdcm/db_stats.py:    def get_doc_data(self, key):
sdcm/tester.py:            nemesis_stats = self.get_doc_data(key='nemesis')
```

Reply:
Let's keep the get_doc_id for code clarity (developer should be able to understand that doc-id = test_id from the code)